sst.config.ts

```diff
@@ -46,6 +48,8 @@ export default {
     app.stack(UcanInvocationStack)
     app.stack(BillingStack)
     app.stack(FilecoinStack)
+    app.stack(ElasticIpfsStack)
+    app.stack(IpniStack)
```

Comment:
deploys are already so long, I would suggest to avoid adding more stacks. It also makes it closer to limits of AWS like roles, etc.

Perhaps a middle ground solution could be to have an `IndexingStack`, where both these would live? by the end of the day, the `ElasticIpfsStack` is just references to other infra and hopefully we can remove that code soon.

Reply:
Hmm, yes indexer stack is a good shout and doesn't tie to IPNI.